src/algebra/group/defs.lean

```diff
@@ -592,39 +645,52 @@ zpow_coe_nat a n
 theorem zpow_neg_succ_of_nat (a : G) (n : ℕ) : a ^ -[1+n] = (a ^ (n + 1))⁻¹ :=
 by { rw ← zpow_coe_nat, exact div_inv_monoid.zpow_neg' n a }
 
-end
+@[to_additive] lemma div_eq_mul_inv (a b : G) : a / b = a * b⁻¹ := div_inv_monoid.div_eq_mul_inv _ _
```

Comment:
The two versions of `div_eq_mul_inv` are exactly the same, right? So we could just as well do:
```suggestion
export div_inv_monoid (div_eq_mul_inv) sub_neg_monoid (sub_eq_add_neg)
```
(And then they'd be guaranteed to stay the same.)

Reply:
I did not change this. Doesn't that mess up the way they are shown in the documentation (it won't be shown under the shorter name and won't have its standalone declaration)? I believe this is why the lemma is manually copied in the root namespace, for discoverability.

Reply:
> I believe this is why the lemma is manually copied in the root namespace, for discoverability.

I wrote the original version, and there the reason was that I wasn't thinking about `export` when I modified `group_with_zero.div_eq_mul_inv`.

In any case, there are more than enough typeclasses where the fields are exported so at best this is an argument for changing doc-gen to handle that better.